arkane/ess/qchem.py

```diff
@@ -380,17 +380,27 @@ def load_negative_frequency(self):
         Return the imaginary frequency from a transition state frequency
         calculation in cm^-1.
         """
-        frequency = 0
+        read_freqs = False
+        num_freq_blocks = 0
         with open(self.path, 'r') as f:
             for line in f:
-                # Read imaginary frequency
-                if ' Frequency:' in line:
-                    frequency = float((line.split()[1]))
-                    break
-        # Make sure the frequency is imaginary:
-        if frequency < 0:
-            return frequency
-        else:
+                # only read the first row from a frequency block
+                if read_freqs and ' Frequency:' in line:
+                    freqs = np.array([float(freq) for freq in line.split()[1:4]])
+                    num_freq_blocks += 1
+                    read_freqs = False
+
+                if 'VIBRATIONAL ANALYSIS' in line:
+                    read_freqs = True
+
+        logging.info(f'Identified {num_freq_blocks} frequency block(s)...')
+        logging.info('Only examining frequencies from the last block...')
+        neg_idx = np.where(freqs < 0)[0]
+        if len(neg_idx) == 1:
+            return freqs[neg_idx[0]]
+        elif len(neg_idx) > 1:
+            raise LogError('More than one imaginary frequency in QChem output file {0}.'.format(self.path))
```

Comment:
Do we also raise these errors in other ESS adapters? What if we (rarely) care about a second order TS? Perhaps make this a warning? We could also think of making this behavior uniform throughout the ESS adapters.

Reply:
@alongd Thanks for reviewing! Currently, the ESS adapters only raise a `LogError` if there are 0 negative frequencies in the file. I initially pushed a change that allowed all ESS adapters to log a warning if there is >1 negative frequency. Unfortunately logging warnings causes our functional tests to [crash](https://github.com/ReactionMechanismGenerator/RMG-Py/runs/7063181776?check_suite_focus=true#step:11:2604
) so I changed it to `logging.info`. 

When reviewing, I'd recommend just looking at the "Files changed" tab. To recap my other fixup commits, I initially assumed the user would want to know the multiple negative frequencies for convenience. However, returning multiple negative frequencies is problematic since it makes the return object inconsistent and thus incompatible with portions of `arkane/statemech.py` that expect 1 negative frequency.

If needed, discussions about handling more than 1 negative frequency could be moved to a separate PR since the primary goal of this PR is to fix the incorrect parsing from the previous QChemLog in which it would return a frequency from an unoptimized geometry.